docs/content/2.usage/1.nuxt-img.md

```diff
@@ -310,6 +312,12 @@ const nonce = useNonce()
 </script>
 ```
 
+## Slots
+
+### `placeholder`
+
+Use this slot to display a placeholder before the actual image loads
```

Comment:
suggestion: maybe we could add an actual code example how to use this slot and how a placeholder could look like (it can be very simple one liner but still something that users can easily copy and paste into their project and adjust later on)

Reply:
I added a snippet

```html
<NuxtImg
  src="/images/colors.jpg"
>
  <template #placeholder>
    I am visible before the image loads!
  </template>
</NuxtImg>
```